presto-manager-agent/src/main/java/com/teradata/prestomanager/agent/PrestoUninstaller.java

```diff
@@ -59,14 +58,14 @@ private static void uninstallUsingRpm(String packageName, boolean checkDependenc
             if (!ignoreErrors) {
                 throw new PrestoManagerException(format("Package %s is not installed", packageName), checkPackageInstalled);
             }
-            LOGGER.warn("Package {} is not installed; Process exited with return value: {}", packageName, checkPackageInstalled);
+            LOGGER.warn("Package '%s' is not installed; Process exited with return value: %s", packageName, checkPackageInstalled);
             return;
         }
         if (executeCommand("service", "presto", "status") == 0) {
             if (!ignoreErrors) {
                 throw new PrestoManagerException("Presto is running");
             }
-            LOGGER.warn("Presto is running");
+            LOGGER.warn("Attempted to uninstall, but Presto is running");
```

Comment:
At this stage the uninstall process will continue to happen even if Presto is running. I think the error message should be `Attempting to uninstall despite Presto running` or something like that.